common/table.js

```diff
@@ -100,6 +100,7 @@
         */
         function read(scope, isBackground) {
 
+            if (scope.vm.search == '') scope.vm.search = null;
```

Comment:
This is done so that the check on line 351 can compare like values. When the user uses back space to clear the search term, the search term field is `''` and the one on the new location object is set to `null` when it sees empty string as a search term.